server/graphql/v2/mutation/AddFundsMutations.ts

```diff
@@ -84,6 +86,10 @@ export const addFundsMutation = {
       );
     }
 
+    if (get(account, `data.features.${FEATURE.ALL}`) === false) {
```

Comment:
Maybe we should add an helper method `account.isFrozen()` for that to make the code clearer.